### PR TITLE
Ci clang

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: .github/workflows/build_test.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: zephyrprojectrtos/ci:v0.26.4
+    env:
+      CMAKE_PREFIX_PATH: /opt/toolchains
+    strategy:
+      matrix:
+        plaform: [rcar_spider]
```
Comment (al1img) [inline .github/workflows/build_test.yaml:22]: we should add gen3 board as well

File: tests/ocispec/src/main.cpp [hunk 2]
```diff
@@ -104,7 +104,7 @@ ZTEST_SUITE(ocispec, NULL, Setup, NULL, NULL, Cleanup);
  * Tests
  **********************************************************************************************************************/
 
-ZTEST(ocispec, ImageManifest)
+ZTEST(ocispec, test_ImageManifest)
```
Comment (al1img) [inline tests/ocispec/src/main.cpp:107]: fix commit message 'testr'

File: README.md [hunk 3]
```diff
@@ -30,14 +30,17 @@ west zephyr-export
...
 west build -b ${BOARD} -p auto
-west build -t fix_dtb
 west build -t run
 ```
 
 For test and debug purpose `native_posix_64` or `native_posix` board can be used.
 For simulation `qemu_x86` or `qemu_x86_64` board can be used.
 For xen based system `xenvm-qemu` board can be used.
 
+For `rcar_spider` and `rcar_salvator_xs_m3`
```
[the remark this answers is not in the document]
Comment (Eabramov84) [reply README.md:43]: fixed

File: .github/workflows/build_test.yaml (hunk 1 above)
Comment (Eabramov84) [reply .github/workflows/build_test.yaml:22]: done

File: tests/ocispec/src/main.cpp (hunk 2 above)
Comment (Eabramov84) [reply tests/ocispec/src/main.cpp:107]: done

**Review by al1img — APPROVED** [review]

**Pull request merged**